src/libs/actions/Network.js

```diff
@@ -8,6 +8,14 @@ function setIsOffline(isOffline) {
     Onyx.merge(ONYXKEYS.NETWORK, {isOffline});
 }
 
+/**
+ *
+ * @param {Boolean} shouldForceOffline
+ */
+function setShouldForceOffline(shouldForceOffline) {
+    Onyx.merge(ONYXKEYS.NETWORK, {shouldForceOffline, isOffline: shouldForceOffline});
```

Comment:
Updating the value of `isOffline` here stuck out to me. Mainly, it seems like it would leave the network in an incorrect offline state after you switch off "forced offline" mode again - instead of the "real" state of the network we'd just have `isOffline: false`.

One idea to improve... if we're setting `shouldForceOffline` to `false` then we should update `isOffline` with the "real" value after a call to `NetInfo.fetch()`.